apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -212,7 +213,7 @@ export const ProgressDot = Radium(React.createClass({
     const isPeerReview = level.kind === LevelKind.peer_review;
     // Account for both the level based concept of locked, and the progress based concept.
     const isLocked = levelStatus === LevelStatus.locked;
-    const iconForLevelStatus = (isLocked || showLevelName) && !isUnplugged &&
+    const iconForLevelStatus = (isLocked || showLevelName) &&
```

Comment:
I'm not sure why `!isUnplugged` was needed before (possibly just a relic of the code churn here over time). `showLevelName` will be true iff Level.kind is `named_level` or `peer_review`, which means Level.kind will not be `unplugged` and `isUnplugged` will be false. There should also be no way to have a locked unplugged level, so I'm pretty sure this extra check is gratuitous.